x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -639,13 +657,21 @@ function generateNewAndRecoveredInstanceEvents<
         ? `actionGroup(subgroup): '${actionGroup}(${actionSubgroup})'`
         : `actionGroup: '${actionGroup}'`
     }`;
-    logInstanceEvent(id, EVENT_LOG_ACTIONS.activeInstance, message, actionGroup, actionSubgroup);
+    logInstanceEvent(
+      id,
+      EVENT_LOG_ACTIONS.activeInstance,
+      message,
+      ruleTypeId,
+      actionGroup,
+      actionSubgroup
+    );
   }
 
   function logInstanceEvent(
     instanceId: string,
     action: string,
     message: string,
+    ruleId: string,
```

Comment:
Why the variable name is `ruleId` instead of `ruleTypeId` as it is above?

Reply:
This is an inner function within the scope of the `generateNewAndRecoveredInstanceEvents` function, which has `ruleTypeId` as a variable so I was getting typescript warnings about a shadowed variable. I agree `ruleId` is confusing though...what about just `ruleType`?

Reply:
Lets keep it as it is 👍